ci/docker/conda-r.dockerfile

```diff
@@ -32,15 +32,15 @@ RUN conda install -q \
 # Ensure parallel compilation of each individual package
 RUN printf "\nMAKEFLAGS=-j8\n" >> /opt/conda/lib/R/etc/Makeconf
 
-# Arrow build flags
-ENV ARROW_FLIGHT=OFF \
+ENV \
```

Comment:
Nit, just hurts my eyes :)

Reply:
But it's easier to add / remove entries.